Mathlib/Topology/ContinuousFunction/CompactlySupported.lean

```diff
@@ -0,0 +1,709 @@
+/-
+Copyright (c) 2024 Yoh Tanimoto. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yoh Tanimoto
+-/
+import Mathlib.Topology.ContinuousFunction.Bounded
+import Mathlib.Topology.ContinuousFunction.CocompactMap
+import Mathlib.Topology.ContinuousFunction.Compact
+import Mathlib.Topology.ContinuousFunction.ZeroAtInfty
+
+/-!
+# Compactly supported continuous functions
+
+The type of compactly supported continuous functions. When the domain is compact,
+`C(α, β) ≃ C_c(α, β)` via the identity map. When the codomain is a metric space, every continuous
+compactly supported map is a bounded continuous function. When the domain is a locally
+compact space, this type has nice properties.
```

Comment:
Could you be a little bit more specific in the file-level docstring, with a short introduction saying what the file is about, and then a section introduced with `## Main definitions and results` where you give the name of the main statements, and explain their content? Otherwise, someone just opening the file will need to browse through the whole file to find the name of the statements, and won't have any pointer to help.

Reply:
I made the description a bit more specific. I think the main points of this file are the definitions and instances, and I wrote so.